### ISSUE TITLE
Owned Types for Query Types

### ISSUE THREAD

**Issue by yringler** [aspnet/EntityFramework.Docs#1245]
I don't see any hint either in this document, in the [owned entities documentation]( https://docs.microsoft.com/en-us/ef/core/modeling/owned-entities), or in the [raw sql documentation](https://docs.microsoft.com/en-us/ef/core/querying/raw-sql)  that query types don't support owned types, but QueryTypeBuilder does not have any methods from the Owns(.*) family.

I think this absence should be stated clearly somewhere.

Is there an open issue to add owned types to query types? Is there a way to use owned types on stored procedure mappings? Should I declare it with `modelBuilder.Entity<T>()`?

Yes, relationships aren't supported for stored procedures, but owned types are also (and I would guess, mostly) about mapping results from the given query to nested properties, not just other tables.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ace630fc-7310-10b7-39bb-26fcfe3f3950
* Version Independent ID: dbe96b32-0589-0f8c-6558-11a065d84dff
* Content: [Query Types - EF Core](https://docs.microsoft.com/en-us/ef/core/modeling/query-types)
* Content Source: [entity-framework/core/modeling/query-types.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/modeling/query-types.md)
* Product: **entity-framework**
* GitHub Login: @anpete
* Microsoft Alias: **divega**

**Comment by mhoerandner** [comment]
I'd also like to use **OwnedTypes** for **QueryTypes** either.

**Comment by yringler** [comment]
See [this issue](https://github.com/aspnet/EntityFrameworkCore/issues/14497).
TL;DR: owned types won't be coming to query types, but maybe eventually to entity types.

**Comment by AndriySvyryd** [comment]
Query types have been renamed to keyless entity types.

For what scenarios would you need a keyless entity type with an owned type as opposed to another keyless entity type?

**Comment by AndriySvyryd** [comment]
Fixed in 75d9b06ee6b0758eb6d7720bcb2dc4f985974060
